storage/state-view/src/lib.rs

```diff
@@ -47,6 +51,10 @@ pub trait TStateView {
 
     /// Get state storage usage info at epoch ending.
     fn get_usage(&self) -> Result<StateStorageUsage>;
+
+    fn as_in_memory_state_view(&self) -> InMemoryStateView {
+        unreachable!("in-memory state view conversion not supported yet")
+    }
```

Comment:
hmm.. if possible I'd avoid adding this to the trait.

Reply:
I can't think of any cleaner way to do this. Any suggestion? :)

Reply:
Maybe isolate it to a `triat AsInMemoryStateview`? only the client will require this trait.

Reply:
Tried this out and this doesn't work unfortunately.

```
error[E0119]: conflicting implementations of trait `in_memory_state_view::AsInMemoryStateView` for type `in_memory_state_view::InMemoryStateView`
  --> storage/state-view/src/in_memory_state_view.rs:51:1
   |
17 | impl<T: StateView> AsInMemoryStateView for T {
   | -------------------------------------------- first implementation here
...
51 | impl AsInMemoryStateView for InMemoryStateView {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `in_memory_state_view::InMemoryStateView`
   ```